ext/proxy/Dxxxx.md

```diff
@@ -289,8 +289,8 @@ template <Readable I1, Readable I2>
   requires !Swappable<reference_t<I1>, reference_t<I2>>() &&
     IndirectlyMovable<I1, I2>() && IndirectlyMovable<I2, I1>()
```

Comment:
> Am I understanding correctly that you're dropping the definitions of IndirectlyMovable at al. herein and referring to those in N4569?

Yes. The technical specifications reflects that but the preceding discussion does not. I can fix that.